RELEASE.md

```diff
@@ -0,0 +1,230 @@
+# [v3.4.0] Policy Refresh

```

Comment:
Please can you put this content in the wiki as well? Needs a page like the others and then putting in the sidebar. Thanks